.gitpod.yml

```diff
@@ -6,19 +6,40 @@ github:
     branches: true
     # enable for pull requests coming from this repo (defaults to true)
     pullRequests: true
+    # enable for pull requests coming from forks (defaults to false)
+    pullRequestsFromForks: true
     # add a "Review in Gitpod" button as a comment to pull requests (defaults to false)
     addComment: true
     # configure whether Gitpod registers itself as a status check to pull requests
     addCheck: false
 
 tasks:
   - name: prebuild
-    init: ./scripts/start.sh && ./scripts/stop.sh && gp sync-done prebuild
-    command: curl -L --http1.1 https://cnfl.io/ccloud-cli | sudo sh -s -- -b /usr/local/bin; exit
+    init: |
+      cd .. && git clone https://github.com/confluentinc/jmx-monitoring-stacks && cd cp-demo/. && \
+        ./scripts/start.sh && ./scripts/stop.sh && \
+        gp sync-done prebuild
+    command: |
+      curl -L --http1.1 https://cnfl.io/ccloud-cli | \
+        sudo sh -s -- -b /usr/local/bin; exit
 
   - name: cp-demo
     init: gp sync-await prebuild
-    command: if [ -z "$DISABLE_AUTOSTART" ]; then echo "🚀 Starting up cp-demo (you can disable autostart by exporting DISABLE_AUTOSTART environment variable, see https://www.gitpod.io/docs/environment-variables)";./scripts/start.sh; echo "🚀 You can now follow steps in https://docs.confluent.io/platform/current/tutorials/cp-demo/docs/on-prem.html#guided-tutorial"; else echo "ℹ️ DISABLE_AUTOSTART environment variable is set, use ./scripts/start.sh to start cp-demo";fi
+    command: |
+      if [ -z "$DISABLE_AUTOSTART" ]; then
+        echo "🚀 Starting up cp-demo (you can disable autostart by exporting DISABLE_AUTOSTART environment variable, see https://www.gitpod.io/docs/environment-variables)";
+        if [ -z "$MONITORING_STACK" ]; then
+          echo "cp-demo is not including any JMX monitoring stack (you can enable them by exporting MONITORING_STACK environment variable, see https://github.com/confluentinc/jmx-monitoring-stacks)."
+          ./scripts/start.sh;
+        else
+          echo "cp-demo includes JMX monitoring stack $MONITORING_STACK"
+          cd ../jmx-monitoring-stacks
+          ./$MONITORING_STACK/start.sh;
```

Comment:
I would suggest the JMX repo also set `CLEAN=true` when starting CP Demo in its own start scripts. My thinking is it's possible for a prebuild to create certificates and junk that should probably be regenerated.